message GyroscopeReading {
   required float x = 1;
   required float y = 2;
   required float z = 3;
   required fixed64 timestamp_millis_unix_epoch = 4;
}
